### PR TITLE
adding archive release

### PR DESCRIPTION
**Pull request by antgonza**
This generates the archive release; basically it allows the plugins to run the given command to generate archiving releases. Additionally we are adding a new link to download those releases. 

Now, the menu looks like:
screen shot 2018-12-20 at 9 22 25 am

### CODE REVIEW

File: qiita_db/test/test_archive.py [hunk 1]
```diff
@@ -63,6 +66,11 @@ def test_insert_from_biom_and_retrieve_feature_values(self):
         obs = qdb.archive.Archive.retrieve_feature_values('Nothing')
         self.assertEqual(obs, exp)
 
+        # now merging_schemes should have 3 elements
+        self.assertDictEqual(qdb.archive.Archive.merging_schemes(), {
+            1: 'Pick closed-reference OTUs | Split libraries FASTQ',
+            2: '', 3: 'Single Rarefaction | N/A'})
```
Comment (charles-cowart) [inline qiita_db/test/test_archive.py]: is 2: '' missing text?

File: qiita_db/handlers/archive.py [hunk 2]
```diff
@@ -57,5 +57,4 @@ def patch(self):
 
         ms = Archive.get_merging_scheme_from_job(ProcessingJob(req_path))
 
-        archive = Archive()
-        self.write(archive.insert_features(ms, loads(req_value)))
+        self.write(Archive.insert_features(ms, loads(req_value)))
```
[the remark this answers is not in the document]
Comment (charles-cowart) [reply qiita_db/handlers/archive.py]: that's funny, I thought I checked that...

File: qiita_db/archive.py [hunk 3]
```diff
@@ -153,9 +172,9 @@ def retrieve_feature_values(cls, archive_merging_scheme=None,
             else:
                 qdb.sql_connection.TRN.add(sql.format(''))
 
-            return {k: v for k, v in
-                    qdb.sql_connection.TRN.execute_fetchindex()}
+            return dict(qdb.sql_connection.TRN.execute_fetchindex())
 
+    @classmethod
     def insert_features(self, merging_scheme, features):
```
Comment (charles-cowart) [inline qiita_db/archive.py]: I hate to sound picky, but if insert_features is going to be a class method, can you change all instances of self to cls, just for convention's sake?

File: qiita_db/handlers/archive.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (charles-cowart) [reply qiita_db/handlers/archive.py]: Ah I see, the @classmethod annotation is new.

File: qiita_db/archive.py (hunk 3 above)
Comment (antgonza) [reply qiita_db/archive.py]: done.

Comment (charles-cowart) [reply qiita_db/archive.py]: thanks! Will merge as soon as testing is completed.

**Pull request merged**